uploadNeuroDB/minc_insertion.pl

```diff
@@ -96,7 +100,11 @@
                  ["General options","section"],
                  ["-verbose", "boolean", 1, \$verbose, "Be verbose."],
            
+                 ["-acquisition_protocol","string", 1, \$acquisitionProtocol,
+                  "Suggest the acquisition protocol to use."],
 
+                 ["-create_minc_pics", "boolean", 1, \$create_minc_pics,
+                  "Use this if invoked directly witouth tarchiveloader."],
```

Comment:
The help should describe what the flag does, not when to use it. ("without" is also spelled wrong.)